plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -184,6 +192,18 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
         makeBuiltinMeaning
             BS.drop
             (runCostingFunTwoArguments . paramDropByteString)
+    toBuiltinMeaning SliceByteString =
+        makeBuiltinMeaning
+            (\from to xs -> BS.take (to - from + 1) (BS.drop from xs))
```

Comment:
I wonder if we should have some tests for this (although maybe we do and I just missed them)?  I always have to think about this kind of thing quite carefully to convince myself it's correct and not off by one somewhere. (And I did think about it and I'm sure it is correct, but maybe the machine should check too).

Reply:
I was thinking about to suggest to move the implementations from here to somewhere else because there are duplicate code in `Builtins/Internal.hs` now. And tests would be nice.